src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
> You could ponder disallowing doing it explicitly and do an inverse of that when parsing.

I don't know if it's possible to do the inverse because it is actually a valid possibility in the case of a "redeem" payment where you actually are paying to the issuer.